index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta lang="ru">
+    <meta charset="UTF-8">
+    <link href="blocks/header/header.css" rel="stylesheet">
+    <link href="blocks/header/__burger-menu/header__burger-menu.css" rel="stylesheet">
+    <link href="blocks/header/__navigation/header__navigation.css" rel="stylesheet">
+    <link href="blocks/common/button/button.css" rel="stylesheet">
+    <link href="blocks/common/support/support.css" rel="stylesheet">
+    <link href="blocks/common/title/title.css" rel="stylesheet">
+    <link href="blocks/main/main.css" rel="stylesheets">
+    <link href="blocks/catalog/catalog.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100&display=swap"
+        rel="stylesheet">
+    <title>Морские прогулки</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header__burger-menu">
+            <img class="burger-menu__icon" src="assets/img/header/menu.svg"
+                alt="Иконка бургер меню, при нажатии на которую открывается поп-ап с меню">
+        </div>
+        <nav class="header__navigation navigation">
+            <div class="navigation__item navigation__item_account">Личный кабинет</div>
+            <div class="navigation__item navigation__item_currency">Рубль</div>
+            <div class="navigation__item navigation__item_language">Ru</div>
+        
+        </nav>
+        <div class="support">
+            <div class="support__phone-number">8 800 500-61-05</div>
+        </div>
+        <div class="header__logo"><img   src="assets/img/header/logo.svg" alt="Логотип сайта"></div>
+        <div class="header__title title">Морские прогулки</div>   
+        <div class="header__subtitle">ПО ЧЁРНОМУ МОРЮ</div>       

```

Comment:
It's always better to control text styles with css. So let's consider using `text-transform` instead of placing caps lock words inside html.